extensions/ql-vscode/src/view/remote-queries/ViewTitle.tsx

```diff
@@ -1,9 +1,13 @@
 import styled from 'styled-components';
```

Comment:
Is it worth moving this file into a shared location (`view/common` rather than having some stuff in remote-queries and some in `variant-analysis`)? I'm happy for this to be in a subsequent PR if we agree it's the right thing to do.

Reply:
👍, I think that would be very useful. We can also consider making these importable by just specifying the `../common` when we place an `index.ts` file inside the `common` directory.

Reply:
Great! Are you happy to do that as part of the issue you're working on, as a separate PR?

Reply:
It might be better to do that in a separate PR. There are some other components which we might also be able to move to the `common` directory such as `VerticalSpace`, so the changeset might get quite large (especially since all imports also need to be changed).

Reply:
Yeah I'd definitely prefer to see it in a separate PR!